packages/engine/Source/Scene/Azure2DImageryProvider.js

```diff
@@ -13,16 +13,15 @@ const trailingSlashRegex = /\/$/;
  *
  * Initialization options for the Azure2DImageryProvider constructor
  *
- * @property {object} options Object with the following properties:
- * @property {string} [options.url="https://atlas.microsoft.com/"] The Azure server url.
- * @property {string} options.tilesetId="microsoft.imagery" The Azure tileset ID. Valid options are {@link microsoft.imagery}, {@link microsoft.base.road}, and {@link microsoft.base.labels.road}
- * @property {string} options.subscriptionKey The public subscription key for the imagery.
- * @property {Ellipsoid} [options.ellipsoid=Ellipsoid.default] The ellipsoid.  If not specified, the default ellipsoid is used.
- * @property {number} [options.minimumLevel=0] The minimum level-of-detail supported by the imagery provider.  Take care when specifying
+ * @property {string} [url="https://atlas.microsoft.com/"] The Azure server url.
+ * @property {string} tilesetId="microsoft.imagery" The Azure tileset ID. Valid options are {@link microsoft.imagery}, {@link microsoft.base.road}, and {@link microsoft.base.labels.road}
+ * @property {string} subscriptionKey The public subscription key for the imagery.
```

Comment:
If I am to be very picky, I think the required property should be moved to the first item in the list of properties.

That suggestion does not need to hold up this PR from being merged.